cms/djangoapps/contentstore/tasks.py

```diff
@@ -47,10 +51,243 @@
 from xmodule.modulestore.exceptions import DuplicateCourseError, ItemNotFoundError
 from xmodule.modulestore.xml_exporter import export_course_to_xml, export_library_to_xml
 from xmodule.modulestore.xml_importer import import_course_from_xml, import_library_from_xml
+from xmodule.video_module.transcripts_utils import (
+    Transcript,
+    clean_video_id,
+    get_transcript_from_contentstore,
+    TranscriptsGenerationException
+)
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.exceptions import NotFoundError
+from edxval.api import (
+    ValCannotCreateError,
+    create_video_transcript,
+    is_transcript_available,
+    create_or_update_video_transcript,
+    create_external_video,
+)
+
+User = get_user_model()
 
 LOGGER = get_task_logger(__name__)
 FILE_READ_CHUNK = 1024  # bytes
 FULL_COURSE_REINDEX_THRESHOLD = 1
```

Comment:
above two variables are unused?

Reply:
these are from the branched off master code ...